SPECS/libnvidia-container/libnvidia-container.spec

```diff
@@ -0,0 +1,278 @@
+BuildRequires:  make
+BuildRequires:  rpcsvc-proto
+BuildRequires:  which
+
+%description
+The nvidia-container library provides an interface to configure GNU/Linux
+containers leveraging NVIDIA hardware. The implementation relies on several
+kernel subsystems and is designed to be agnostic of the container runtime.
+
+%prep
+%setup -q
+%patch0 -p1
+%patch1 -p1
+
+mkdir -p deps/src/nvidia-modprobe-%{modprobe_version}
+tar -C deps/src/nvidia-modprobe-%{modprobe_version} --strip-components=1 -xzf %{SOURCE1}
+%patch2 -p1 -d deps/src/nvidia-modprobe-%{modprobe_version}
+touch deps/src/nvidia-modprobe-%{modprobe_version}/.download_stamp
+
+%build
+%make_build WITH_LIBELF=yes
+
+%install
+DESTDIR=%{buildroot} make install prefix=%{_prefix} \
+	exec_prefix=%{_prefix} \
+	bindir=%{_bindir} \
+	libdir=%{_libdir} \
+	includedir=%{_includedir}\
+	docdir=%{_licensedir} \
+	WITH_LIBELF=yes
+
+%package -n %{name}%{_major}
+Summary:        NVIDIA container runtime library
+
+%description -n %{name}%{_major}
+The nvidia-container library provides an interface to configure GNU/Linux
+containers leveraging NVIDIA hardware. The implementation relies on several
+kernel subsystems and is designed to be agnostic of the container runtime.
+
+This package requires the NVIDIA driver (>= 340.29) to be installed separately.
+
+%package devel
+Summary:        NVIDIA container runtime library (development files)
+Requires:       %{name}%{_major}%{?_isa} = %{version}-%{release}
+
+%description devel
+The nvidia-container library provides an interface to configure GNU/Linux
+containers leveraging NVIDIA hardware. The implementation relies on several
+kernel subsystems and is designed to be agnostic of the container runtime.
+
+This package contains the files required to compile programs with the library.
+
+%package static
+Summary:        NVIDIA container runtime library (static library)
+Requires:       %{name}-devel%{?_isa} = %{version}-%{release}
+
+%description static
+The nvidia-container library provides an interface to configure GNU/Linux
+containers leveraging NVIDIA hardware. The implementation relies on several
+kernel subsystems and is designed to be agnostic of the container runtime.
+
+This package requires the NVIDIA driver (>= 340.29) to be installed separately.
+
+%define debug_package %{nil}
+
+%package -n %{name}%{_major}-debuginfo
+Summary:        NVIDIA container runtime library (debugging symbols)
+Requires:       %{name}%{_major}%{?_isa} = %{version}-%{release}
+
+%description -n %{name}%{_major}-debuginfo
+The nvidia-container library provides an interface to configure GNU/Linux
+containers leveraging NVIDIA hardware. The implementation relies on several
+kernel subsystems and is designed to be agnostic of the container runtime.
+
+This package contains the debugging symbols for the library.
+
+%package tools
+Summary:        NVIDIA container runtime library (command-line tools)
+Requires:       %{name}%{_major}%{?_isa} >= %{version}-%{release}
```

Comment:
Is the `>=` on purpose not a `=`?

Reply:
That was a mistake on my end. this will be fixed in the next push